extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -76,6 +76,15 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		if (bookPathToTrust) {
 			let trustChanged = this._bookTrustManager.setBookAsTrusted(bookPathToTrust);
 
+			// update trust state of opened items
+			if (azdata.nb.notebookDocuments && azdata.nb.notebookDocuments.length) {
+				azdata.nb.notebookDocuments.forEach(document => {
+					if (this._bookTrustManager.isNotebookTrustedByDefault(document.uri.fsPath)) {
+						document.setTrusted(true);
```

Comment:
Shouldn't you only do this for notebooks within the book that was just trusted?

Reply:
Yes, you are right.
# resolved